src/model/operation/transform.js

```diff
@@ -401,7 +403,7 @@ class ContextFactory {
 		// gathered during transformation that we want to save for given operation, is actually saved for the original operation.
 		// This way no matter if operation `a` is cloned, then transformed, even breaks, we still have access to the previously
 		// gathered data through original operation reference.
-		this._originalOperations = new Map();
```

Comment:
Public properties should be moved above the private ones.

Reply:
Done.